sroka/api/s3_connection/s3_connection_api.py

```diff
@@ -28,15 +29,25 @@ def _download_data(key_prefix, s3, bucket_name, prefix, sep):
             df_list.append(pd.read_csv(data, error_bad_lines=False, warn_bad_lines=False, sep=sep))
         except UnicodeDecodeError:
             df_list.append(pq.read_pandas(data).to_pandas())
+        except EmptyDataError:
+            print('File is empty')
```

Comment:
I don't know the `file` class in Python and if it has a method returning the file name I suggest printing also the name of an empty file.

Reply:
This message is printed only in case a single file is downloaded, it would print exactly the same argument that was passed to the function.